src/Atmos/Model/ref_state.jl

```diff
@@ -17,6 +16,19 @@ condition or for linearization.
 """
 abstract type ReferenceState end
 
+"""
+    TemperatureProfile
+
+Specifies the temperature profile for a reference state.
+
+Instances of this type are required to be callable objects with the following signature
+
+    T,p = (::TemperatureProfile)(orientation::Orientation, aux::Vars)
+
+where `T` is the temperature (in K), and `p` is the pressure (in hPa).
```

Comment:
```suggestion
where `T` is the temperature or virtual temperature (in K), and `p` is the pressure (in hPa).
```

Reply:
I did a search-all, only saw `hPa` here and in EDMF documentation, which I just removed since it was a bit more of a reference at the time